bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/DialogProcessor.java

```diff
@@ -309,7 +309,7 @@ private void toggleProcessing(boolean value) {
         processing = value;
         String item = dialogContext.listeningItem();
         if (item != null && ItemUtil.isValidItemName(item)) {
-            OnOffType command = (value) ? OnOffType.ON : OnOffType.OFF;
+            OnOffType command = value ? OnOffType.ON : OnOffType.OFF;
```

Comment:
There is a `from` method for this nowadays:

```suggestion
            OnOffType command = OnOffType.from(value);
```